PrioritizationModel/PriotizationModelController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Extensions.Configuration;
+using DataAccessLayer.Interfaces;
+using DataAccessLayer.Models;
+using PrioritizationService.DTOs;
+using WebAPI.DTOs.DTOConverters;
+
+namespace PrioritizationModel;
+
+public class PrioritizationModelController
+{
+    private readonly IAssetDataAccess _assetDataAccess;
+    private readonly IPrioritizationModel _prioritizationModel;
+    private IEnumerable<AssetDTO> _assets;
+
+    public PrioritizationModelController(IAssetDataAccess assetDataAccess, IPrioritizationModel prioritizationModel)
```

Comment:
I wonder if it is better to provide a connection string and the model controller gets the data access trough the factory itself
or to keep it like that and get the data access in the Program.cs of the WebAPI and pass it when adding services
_aka services.AddScoped(...)_
This would also remove the need to give configuration to the method below since that is used to get the connection string only.

Reply:
Solved with: [7d2de0d](https://github.com/diversiwaver/aFRR-Prioritization-Service/pull/9/commits/7d2de0d1e6e36e59e20c05cadc7ce81ee2216a11)